build/lib/pgBackRestBuild/Config/Data.pm

```diff
@@ -1748,6 +1760,82 @@ my %hConfigDefine =
         }
     },
 
+    &CFGOPT_REPO_AZURE_ACCOUNT =>
+    {
+        &CFGDEF_SECTION => CFGDEF_SECTION_GLOBAL,
+        &CFGDEF_TYPE => CFGDEF_TYPE_STRING,
+        &CFGDEF_PREFIX => CFGDEF_PREFIX_REPO,
+        &CFGDEF_INDEX_TOTAL => CFGDEF_INDEX_REPO,
+        &CFGDEF_SECURE => true,
```

Comment:
Because the account name leaks details of the company/organization running pgbackrest. In general we would like the logs to be free of those sorts of details. You'll still see them at trace level because they get output in the URIs, but that's just the deal with trace level.

I did forget to use FUNCTION_TEST_PARAM() a few places where we don't want to log keys so go ahead and do that.